src/components/ButtonAdd.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { View, TouchableOpacity } from 'react-native';
+
+import { Ionicons } from '@expo/vector-icons';
+
+import Styles from '../styles/MyFlights';
+
+export default class ButtonAdd extends Component {
+  render() {
+    return (
+      <View>
```

Comment:
The `<View>` component should be removed since it doesn't add any style

Reply:
Changes done